docs/guides/namecheap_domain_records_guide.md

```diff
@@ -34,6 +34,9 @@ The same workflow works for both: `record` items and `nameservers`.
 
 -> This is the default behavior, however, we recommend to set the mode explicitly.
 
+~> Upon creating a new resource with the records or nameservers that already exist, the provider will throw a duplicate
```

Comment:
I think we need to add that it happens if subdomain & record type match

Reply:
Noted. I will check this.
At the moment, we have duplicates comparison only by `hostname` (subdomain), `type`, and `address`.

However, user allowed to create "mistakes" like:

```terraform
resource "namecheap_domain_records" "domain" {
   #...
   record {
     hostname = "dev"
     type = "A"
     address = "11.22.33.44"
   }

   record {
     hostname = "dev"
     type = "A"
     address = "33.33.33.33"
   }
}
```

I will amend this behavior (and provide tests for all cases! 😆 )